packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -870,7 +880,7 @@ class RenderOpacity extends RenderProxyBox {
     _alpha = ui.Color.getAlphaFromOpacity(_opacity);
     if (didNeedCompositing != alwaysNeedsCompositing)
       markNeedsCompositingBitsUpdate();
-    markNeedsPaint();
+    markNeedsLayerPropertyUpdate();
```

Comment:
should we rename this to `markNeedsCompositedLayerUpdate` to indicate that calling this will cause a call to updateCompositedLayer later on? This also leaves the door open for future improvements where updateCompositedLayer changes the layer type completely.

Reply:
Done